themes/HeroesLounge-Theme/partials/navigation/events.htm

```diff
@@ -0,0 +1,16 @@
+<li class="nav-item dropdown">
```

Comment:
backend/cms is not there to be used to edit anything. So please don't use partials unless it is to avoid code duplication. All this achieves is tech debt where it is completely unclear where the navigation is living.